src/proto/grpc/tls/provider/meshca/meshca.proto

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.security.meshca.v1;
```

Comment:
I think that for now, we should be using the copy of this from the grpc-proto repo:

https://github.com/grpc/grpc-proto/blob/master/grpc/tls/provider/meshca/experimental/config.proto

Reply:
That is different: that's the meshCA config defined as proto (more relevant when it comes from the control plane). Yash needs the proto for MeshCA request/response/service which is not there in grpc-proto AFAIK.

Reply:
Right, Mark, we are talking about different protos. The one that we need here is the proto for the MeshCA service.

Reply:
Doh!  You're right.  I wasn't reading closely.  Sorry for the confusion.

Reply:
For the one that you are referring to is the config.proto and for that I'll probably take the approach that we are already following everywhere else for consistency.